@navikt/core/react/src/util/focus-boundary/FocusBoundary.tsx

```diff
@@ -449,5 +504,25 @@ function removeLinks(items: HTMLElement[]) {
   return items.filter((item) => item.tagName !== "A");
 }
 
+/**
+ * If the provided argument is a ref object, returns its `current` value.
+ * Otherwise, returns the argument itself.
+ *
+ * Non-generic to safely handle refs whose `.current` may be `null`.
+ */
+function resolveRef(
+  maybeRef:
+    | HTMLElement
+    | null
+    | undefined
+    | React.RefObject<HTMLElement | null | undefined>,
+): HTMLElement | null | undefined {
+  if (maybeRef == null) {
+    return maybeRef;
+  }
+
+  return "current" in maybeRef ? maybeRef.current : maybeRef;
+}
```

Comment:
It seems like we don't need to handle null/undefined here
```suggestion
function resolveRef(
  maybeRef: HTMLElement | React.RefObject<HTMLElement | null | undefined>,
): HTMLElement | null | undefined {
  return "current" in maybeRef ? maybeRef.current : maybeRef;
}
```